ratis-server/src/test/java/org/apache/ratis/server/impl/LeaderElectionTests.java

```diff
@@ -436,6 +439,37 @@ void runTestPauseResumeLeaderElection(CLUSTER cluster) throws IOException, Inter
     }
   }
 
+  void runTestPauseResumeLeaderWithProxy(CLUSTER cluster) throws Exception {
```

Comment:
Let's simply update runTestPauseResumeLeaderElection to the code in runTestPauseResumeLeaderWithProxy.